packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -0,0 +1,334 @@
+#import "KeyCodes.h"
+#import "RCTFocusZone.h"
+
+typedef enum {
```

Comment:
nit : is there a limitation here where we can't use the recommended NS_ENUM(NSInteger, FocusZoneAction)  ?

Reply:
Not that I know of.  I avoided changes unless they were specifically called out in the previous PR.

Reply:
seems like a simple fix to try it out?